ci/appveyor-cpp-build.bat

```diff
@@ -132,6 +132,15 @@ set ARROW_HOME=%CONDA_PREFIX%\Library
 @rem ARROW-3075; pkgconfig is broken for Parquet for now
 set PARQUET_HOME=%CONDA_PREFIX%\Library
 
+@rem Download IANA Timezone Database to a non-standard location to
+@rem test the configurability of the timezone database path
```

Comment:
One more question here: we currently also download the database in appveyor-cpp-setup.bat, which is typically always run before this file I would think? 
Should we then remove it there?

Reply:
That part is needed for the C++ tests.
I will keep both database downloads but will remove the folder of the former one after the `ctest` is run.